### PR TITLE
Add configuration

### PR DESCRIPTION
**Pull request by sbryzak**
This PR adds basic configuration to the auth service.

Fixes https://issues.redhat.com/browse/CRT-1624

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,13 @@
 # sandbox-auth
 Authentication Provider Service for Developer Sandbox
+
+## Building
+
+This project requires Podman to build. Refer to the [Podman](https://podman.io/getting-started) getting started guide for installation instructions.
+
+Execute the `build.sh` script to build the Sandbox-Auth container.
```
Comment (alexeykazakov) [inline README.md:8]: Can we have a make target for that to be consistent with the rest of our components?

File: README.md [hunk 2]
```diff
@@ -1,2 +1,13 @@
...
+
+## Building
+
+This project requires Podman to build. Refer to the [Podman](https://podman.io/getting-started) getting started guide for installation instructions.
+
+Execute the `build.sh` script to build the Sandbox-Auth container. 
+
+## Testing
+
+Run the tests by executing `test.sh`. This project uses [Testcontainers for Golang](https://golang.testcontainers.org/) which
```
Comment (alexeykazakov) [inline README.md:12]: Same here.

File: README.md (hunk 1 above)
Comment (sbryzak) [reply README.md:8]: I actually wanted to steer clear of using make, as it now seems to be quite common practice to build using buildah instead in order to achieve a reproducible result no matter where the project is built.  With the build script essentially being a one line command, do we really want to require `make` on top of that?

Comment (xcoulon) [reply README.md:8]: Would you have example of projects that use buildah?
Also, we'll probably have other tasks to run (`lint`, `test`, `test-with-coverage`, etc.). Running them with `make` is pretty helpful.

Comment (sbryzak) [reply README.md:8]: Here's a few articles, one is from medium.com and the other two are from our Red Hat colleagues:

[https://two-oes.medium.com/building-your-first-container-application-in-go-1941919defd7](https://two-oes.medium.com/building-your-first-container-application-in-go-1941919defd7)
[https://www.redhat.com/sysadmin/building-buildah](https://www.redhat.com/sysadmin/building-buildah)
[https://developers.redhat.com/articles/go-container#let_s_get_some_code](https://developers.redhat.com/articles/go-container#let_s_get_some_code)

We can continue discussions about using make, and I'm definitely open to it if it provides us specific benefits.  It's a topic that's not really related to the intent of this PR though, so let's move the discussion to Slack or something.

Comment (xcoulon) [reply README.md:8]: thanks for the links, @sbryzak !

Comment (alexeykazakov) [reply README.md:8]: We can keep using buildah and even `.sh` scripts behind the scenes but let's use make as our interface for all project related actions. It at least will be consistent with other repos. And I like make as a unified tool in general. Let's have the same targets as we have for other repos even if behind the scenes we use different tools.

**Review by alexeykazakov — APPROVED** [review]
I'm approving the PR but please address the makefile comments. It's OK to do it an separate PR.

**Pull request merged**